.github/CODEOWNERS

```diff
@@ -4,3 +4,4 @@
 # review when someone opens a pull request.
 
 * @hewigovens @catenocrypt @milerius
+kotlin/ @MaximPestryakov
```

Comment:
This is not required, please remove - this will assign you every time we open a PR on WC, you do not want that

Reply:
Not every time. Only when kotlin/ folder is affected